packages/vats/src/proposals/network-proposal.js

```diff
@@ -168,6 +168,7 @@ export const getManifestForNetwork = (_powers, { networkRef, ibcRef }) => ({
         zoe: 'zoe',
         provisioning: 'provisioning',
         vatUpgradeInfo: true,
+        portAllocator: 'portAllocator',
```

Comment:
Should this go in `produce` instead of `consume`? Then in `vat-orchestration`, we can consume `portAllocator` instead of `networkVat`?